rayon-core/src/configuration.rs

```diff
@@ -63,7 +63,8 @@ impl Configuration {
     /// Set a closure which takes a thread index and returns
     /// the thread's name.
     pub fn set_thread_name<F>(mut self, closure: F) -> Self
-    where F: FnMut(usize) -> String + 'static {
+        where F: FnMut(usize) -> String + 'static
```

Comment:
It's implemented (rustfmt 8.1) but not the default. where_style = "Rfc" should be it.

Conspicuous config file https://github.com/rust-lang-nursery/rustfmt/blob/master/rfc-rustfmt.toml